product/domain/productBasics.go

```diff
@@ -63,7 +63,9 @@ type (
 
 		CategoryToCodeMapping []string
 
+		// Deprecated: use Stock[x].Level instead
 		StockLevel string
```

Comment:
We should also refactor all usages of this field. see product/infrastructure/fake/productservice.go